tests/topotests/multicast_pim6_dual_stack/test_multicast_pim6_dual_stack.py

```diff
@@ -0,0 +1,1952 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: ISC
+#
+# Copyright (c) 2023 by VMware, Inc. ("VMware")
+# Used Copyright (c) 2018 by Network Device Education Foundation,
+# Inc. ("NetDEF") in this file.
+#
+
+"""
+Following tests are covered to test multicast pim6 dual stack:
+
+1. Verify IPv4 and IPv6 mroute and upstream when mld and MLD join sent from same interface
+   of FRR1 and source present on same interface in FRR4
+2. Verify IPv4 and IPv6 mroute and upstream when mld and MLD join sent from different interface
+   of FRR1 and source present on different interface in FRR4
+3. Verify IPv4 and IPv6 mroute and upstream when mld and MLD join sent from same interface
+   of FRR1 and source present on same interface in FRR4
+4. Verify IPv4 and IPv6 mroute and upstream when mld and MLD join sent from same interface
```

Comment:
Did you mean IGMP and MLD here? Same applies for all other occurences. Please check.

Reply:
yes Mobas, we are enabling both IGMP and MLD for these tests, thats why names as dual stack[v4/v6]. Thanks.

Reply:
What I mean to say here is you need to change "when mld and MLD join" to "when IGMP and MLD join"

Reply:
Updated it to IGMP and MLD. Thanks